custom/icds_reports/migrations/0074_expected_decimal.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.16 on 2018-11-07 16:51
+from __future__ import unicode_literals, absolute_import
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('icds_reports', '0073_ccsrecordmonthly_closed'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='awwincentivereport',
+            name='expected_visits',
+            field=models.DecimalField(decimal_places=2, max_digits=64, null=True),
```

Comment:
There is one more commit coming that I hope will make this clearer (working on tests locally), but I want to keep it as a decimal, and not coerce it to an int with `floor`, because of the new two step generation. Otherwise if the person had 0.44 expected visits from the monthly ucr, they went into agg_ccs_record as 0. and then if they had 0.6 from the cf visits in the normal case ucr, they would combine to be 0.6, and round back to down to 0. If we keep the 0.44 alive at that stage we can add the decimals correct and not round until the end.
In more mathmematical terms `floor(x) + floor(y) != floor(x + y)` the right hand side being what we really want